support-tos.md

```diff
@@ -22,21 +22,40 @@ If specified at time of purchase, a feature key entitles the holder to priority
 
 We may cease providing e-mail support to you at any time for any reason. If we do this, we will immediately notify your primary contact by email and issue a refund for the current billing period. Your feature key will remain valid until its original expiration date, allowing you to continue to use the software without support. Moreover, we will offer you the option to renew your feature key at the same rate but without e-mail support.
 
+## Billing Options
+
+We offer two billing plans:
+
+* **Prepaid annual billing:** You will be charged at the time of purchase for a feature key covering the number of users you specify and valid for 365 days. You may increase the user limit at any time. If you increase the user limit, you will be charged for the new users prorated for the time remaining in the billing cycle. If you decrease the user limit, the change will take effect upon your next renewal.
+
+* **Usage-based monthly billing:** Every 30 days, you will be charged according to the actual number of active users seen by your server in the last 30 days, not counting "visitor" users. Your server reports the number of users automatically. You may inspect your server's active user count by visiting the "statistics" section of the administrator settings. You must not manipulate your server in a way that causes it to report an incorrect count.
```

Comment:
Re: "You must not manipulate your server in a way that causes it to report an incorrect count." Do we want to state what that will result in? Or do we just consider it a "ToS violation"?